node_operations.asciidoc

```diff
@@ -88,9 +94,23 @@ At a minimum, the following will be required to run a Lightning Node:
 
 * **RAM**: a system with 2GB of RAM will _barely_ run both Bitcoin and Lightning nodes. It will perform much better with at least 4GB of RAM. The Initial Block Download will be especially challenging with less than 4GB of RAM. More than 8GB of RAM is unnecessary, because the CPU is the greater bottleneck for these types of services, due to cryptographic operations such as signature validation.
 
-* **Storage Drive**: this can be a Hard Drive or an Solid State Drive (SSD). An SSD will be significantly quicker for running a Bitcoin node. Most of the storage is used for the Bitcoin blockchain, which occupies more than 320GB (as of January 2021).
+* **Storage Drive**: this can be a Hard Disk Drive (HDD) or a Solid State Drive (SSD). 
+An SSD will be significantly quicker (but more expensive) for running a node. 
+Most of the storage is used for the Bitcoin blockchain, which is hundreds of gigabytes large. 
+A fair tradeoff (cost for complexity) is to buy a small SSD to boot the OS from and a larger HDD to store large data objects (mostly databases).
 
-* **Internet Connection**: a reliable internet connection will be required to download new Bitcoin blocks, as well as to communicate with other Lightning peers. During operation the estimated data use ranges from 10GB to 100GB per month, depending on configuration. At startup a Bitcoin full node downloads the full blockchain, 324GB as of January 2021.
+[NOTE]
+====
+Raspberri Pis are a common choice for running node software, due to the cost and parts availability. 
+The OS that runs on the device usually boots from an SD card. 
+For most use-cases, this is a non-issue but Bitcoin Core is notorious for being I/O heavy. 
+This can be problematic if you're running a hot (or Lightning) wallet on the Pi since its running alongside the OS on the SD card. 
+The worst case scenario this can lead to, is that the SD card fails and the data (money) becomes unrecoverable. 
+SATA based storage technology is generally regarded as being more robust and durable than SD cards. 
+In any case, a hardware failure should never result in loss of funds since a backup/recovery strategy should always be present when working with Bitcoin-type sofware.
```

Comment:
but backup with lightnign ist tricky. this sentence reads like: "Don't worry" but I think one should worry about the backup strategy.